analyses/tp53_nf1_score/06-evaluate-classifier.py

```diff
@@ -56,13 +56,25 @@
     dest="outputfile",
     help="output plots basename for TP53 and NF1 ROC curves",
 )
+parser.add_option(
+    "-t", "--apply_threshold",
+    dest = "apply_threshold",
+    action = "store_true",
+    help = "Flag indicating only to consider samples that pass a tumor purity threshold."
+)
 
 (options, args) = parser.parse_args()
 status_file = options.status_file
 scores_file = options.filename
 clinical = options.clinical
 outputfilename = options.outputfile
 
+# Set filename suffix for specifying output TSV filenames
+if options.apply_threshold is True:
+    filename_suffix = "_tumor-purity-threshold"
+else:
+    filename_suffix = ""
+
```

Comment:
I don't think this gets used in `analyses/tp53_nf1_score/run_classifier-tumor_purity_threshold.sh`, and when you call this script in the shell script [you encode the fact that you're using tumor purity filtered data in the output base name](https://github.com/sjspielman/OpenPBTA-analysis/blob/09100897d2890d0a664ab8637e207bb303671096/analyses/tp53_nf1_score/run_classifier-tumor_purity_threshold.sh#L81): 

```
  -o stranded_tumor-purity-threshold
```

So is this (and the associated changes below) required at all?

Reply:
Ah, yes I see I've duplicated this information in output file names! The `--apply_threshold` flag and all that goes with it are indeed not needed - the `filename` input options provides scores from the thresholded analysis as well, so everything is properly specified without needing this.

Therefore, while the `--apply_threshold` flag was indeed missing as noted in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1664/files#r1106490269, it's not going to get added in since it's being removed.